docs/main/user-guides/migration/legacy-storage-to-new-storage/cstor-to-replicated.md

```diff
@@ -0,0 +1,127 @@
+---
+id: cstor-to-replicated
+title: Migration from OpenEBS cStor to OpenEBS Replicated
+keywords:
+ - Migration from OpenEBS cStor to OpenEBS Replicated
+ - cStor to Replicated
+ - cStor to Mayastor
+ - Jiva to Replicated
+ - Jiva to Mayastor
+description: This section outlines the process of migrating OpenEBS cStor to OpenEBS Replicated.
+---
+
+:::info
+The following steps are an example about migrating from legacy storage to new storage.
```

Comment:
'new storage' seems incorrect, need a better phrase.

Reply:
As discussed, we are using the phrase "latest storage solution"